tests/test_ops.py

```diff
@@ -40,6 +40,7 @@ def test_ops_implicit_layout():
         'maxpooling.float32',
         'resize-bilinear.float32',
         'resize-nearest-neighbor.float32',
+        'reshape-transpose.float32',
```

Comment:
What about naming the file as something like `conv-reshape`? We are inserting `Transpose` to fix a `Conv` to `Reshape` issue here.